.env

```diff
@@ -7,4 +7,6 @@ POSTGRES_PORT_NONE=5433
 POSTGRES_PORT_NEON=5434
 POSTGRES_PORT_PGVECTOR=5435
 POSTGRES_PORT_LANTERN=5436
-SSH_PORT_LANTERN=2222
\ No newline at end of file
+SSH_PORT_LANTERN=2222
+PINECONE_API_KEY='YOUR_PINECONE_API_KEY_HERE'
```

Comment:
Coming from JS background so following some conventions from there. What do you think about
1. putting this in .env.local
3. putting .env.local in .gitignore
4. adding a check that these two variables are defined when testing Pinecone
I'm worry about accidentally committing these variables

Reply:
Yes I think moving .env to gitignore and adding .env.example may be an option. Also having checks before using these variables will help to provide better user facing error message